.travis.yml

```diff
@@ -15,12 +15,20 @@ env:
   matrix:
     - MODE=source-checks
     - MODE=test-tools
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_GC=boehm TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_GC=immix TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_GC=commix TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_GC=boehm TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_GC=immix TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_GC=commix TEST_COMMAND="test-runtime"
+    # without the Interflow optimiser
+    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_OPTIMISE=false SCALANATIVE_GC=boehm TEST_COMMAND="test-runtime"
+    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_OPTIMISE=false SCALANATIVE_GC=immix TEST_COMMAND="test-runtime"
+    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_OPTIMISE=false SCALANATIVE_GC=commix TEST_COMMAND="test-runtime"
+    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_OPTIMISE=false SCALANATIVE_GC=boehm TEST_COMMAND="test-runtime"
+    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_OPTIMISE=false SCALANATIVE_GC=immix TEST_COMMAND="test-runtime"
+    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_OPTIMISE=false SCALANATIVE_GC=commix TEST_COMMAND="test-runtime"
```

Comment:
Do you think it's important to test every GC in this configuration? If not I would suggest to only test e.g. immix in the "without the Interflow optimizer" category, for the sake of not exploding the CI time.

Reply:
@LeeTibbert and I had a discussion about doing a sparse matrix of tests hoping that we could add `macOS` back into the mix. It was dropped because there were so many failures related to running of Unit tests on `macOS`.

Reply:
@LeeTibbert and I had a discussion about using a sparse matrix. We previously dropped `macOS` because the `unit-tests` failed to much but it would be great to get that back. We could also add additional architectures like `arm`, `32bit`, Windows or whatever is most important.